packages/contracts-bedrock/test/L1/OptimismPortal.t.sol

```diff
@@ -1060,3 +1125,143 @@ contract OptimismPortalResourceFuzz_Test is CommonTest {
         });
     }
 }
+
+contract OptimismPortalWithMockERC20_Test is OptimismPortal_Test, OptimismPortal_FinalizeWithdrawal_Test {
+    MockERC20 token;
+
+    function setUp() public override(OptimismPortal_Test, OptimismPortal_FinalizeWithdrawal_Test) {
+        OptimismPortal_Test.setUp();
+        OptimismPortal_Test.setUp();
```

Comment:
do you mean to call setup on the 2 contracts instead of calling setup on the same contract twice?

Reply:
yes! this was just fixed